package.json

```diff
@@ -20,7 +20,8 @@
     "i18n:validate": "ttag validate i18n/zh_TW.po",
     "clean": "run-s clean:*",
     "clean:server": "rm -rf build",
-    "clean:liff": "rm -rf liff"
+    "clean:liff": "rm -rf liff",
+    "backtrack": "npm run build:server && USER_ID=1 node -r dotenv/config build/database/backtrack.js"
```

Comment:
1. What does `USER_ID` do? I may have missed where it is used.
2. Since the backtrack will only be run once, I think we can just put a comment to indicate how to run this function instead of putting it in package.json. In reality we may be executing the script using `heroku exec`.

Reply:
1. Use in rumors-line-bot-log-parser to show userId, because we hide the userId by default 
https://github.com/cofacts/rumors-line-bot-log-parser/blob/5446ec86976e4c55f24fd355fe595a45da5f31e2/index.js#L93

2. okok